src/bin/parser_test.rs

```diff
@@ -31,34 +31,35 @@ fn main() -> Result<()> {
         tests_failed: Vec::new(),
     };
 
-    let filenames = Some(&["tests3.dat"][..]);
+    let filenames = Some(&["tests16.dat"][..]);
     let fixtures = testing::tree_construction::fixtures(filenames).expect("fixtures");
 
     for fixture_file in fixtures {
         println!(
-            "🏃‍♂️ Running {} tests from 🗄️ {:?}\n",
+            "\
+            🏃‍♂️ Running {} tests from 🗄️ {:?}",
```

Comment:
Not sure if this is the intended outcome with this formatting:

![2023-10-25_17-20](https://github.com/gosub-browser/gosub-engine/assets/760949/4b1734ca-d096-4814-abf2-62dd6ab395aa)

Might look different on Windows versus Linux.

Reply:
unicode emoticons seems to be displayed a bit off on my terminal. It won't matter much anyway so i'll revert this back